dspace-server-webapp/src/main/java/org/dspace/app/rest/projection/FullProjection.java

```diff
@@ -18,14 +21,17 @@
 public class FullProjection extends AbstractProjection {
 
     public final static String NAME = "full";
+    private final int maxEmbed = DSpaceServicesFactory.getInstance().getConfigurationService()
+            .getIntProperty("projections.full.max", 2);
```

Comment:
This new configuration does not appear to be added to a `*.cfg` file in this PR.  We should add it along with comments describing it.

It should probably be added to the existing `rest.cfg` file, and renamed `rest.projections.full.max`.  (NOTE: this same config is also used in SpecificLevelProjection)